lib/rules/block-no-redundant-nesting/__tests__/index.mjs

```diff
@@ -0,0 +1,113 @@
+import rule from '../index.mjs';
+
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: [true],
+
+	accept: [
+		{
+			code: '',
+		},
+		{
+			code: 'a { color: red; }',
```

Comment:
```suggestion
			code: 'a {}',
```

Let's [follow our conventions](https://stylelint.io/developer-guide/rules#write-tests) for most tests, and a couple of fuller tests towards the end.

> the minimum amount of valid CSS possible, e.g. use an empty rule if targeting selectors)

Let's remove the declarations from the other test cases, too.